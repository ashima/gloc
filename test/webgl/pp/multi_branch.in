#ifdef Q
#else
  #ifdef J
    #if K
    #endif
  #elif defined V
  #elif defined F
  #else
void main() {}
  #endif
#endif
