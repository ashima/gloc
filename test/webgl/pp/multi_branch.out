#line 9
void main() {}
